ovirt-host.spec

```diff
@@ -97,15 +97,18 @@ This meta package pulls in all the dependencies needed for an oVirt hosts.
 Summary:	This meta package pulls in all the dependencies needed for minimal oVirt hosts
 %ifnarch s390x
 # Not available for s390x yet
-Requires:	collectd
-Requires:	collectd-disk
-Requires:	collectd-netlink
-Requires:	collectd-write_http
-Requires:	collectd-virt
-
-%if 0%{?rhel}
-# collectd-write_syslog is available only on RHEL and similar
-Requires:	collectd-write_syslog
+%if %{rhv_build}
+Requires:	collectd >= 5.11.0-10
```

Comment:
Can't we get 5.12 in RHV too? @mrunge any plan to build 5.12 downstream ?

Reply:
We have it downstream.